core/src/main/java/org/ironjacamar/core/CoreLogger.java

```diff
@@ -38,6 +39,40 @@
 @MessageLogger(projectCode = "IJ2")
 public interface CoreLogger extends BasicLogger
```

Comment:
Well, they are a strict porting from IJ 1.x, leave there for future...but maybe it's better to drop them and readd if and when they will be needed